app/Contracts/PhotoCreate/SharedPipe.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\Contracts\PhotoCreate;
+
+use App\DTO\PhotoCreate\DuplicateDTO;
+
+/**
+ * Basic definition of a Photo creation pipe.
```

Comment:
?

Reply:
Good catch. Need to improve the doc here. :D